eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -67,12 +65,12 @@ stages:
                         Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
                       displayName: Stage artifacts
                       timeoutInMinutes: 5
-                    - template: eng/pipelines/templates/scripts/create-tags-and-git-release.yml@azure-sdk-tools
+                    - template: eng/pipelines/templates/steps/create-tags-and-git-release.yml
```

Comment:
I suspect this will only work once you move this under `eng/common`

Reply:
Right